acquia/acquia_cms
# ACMS-960: Added code to save the MAPS key from environment variables.

**Motivation**
* The API key is not set until and unless it is set for Cohesion key settings.
Fixes #NNN

**Proposed changes**
* Once the cohesion keys are set the jobs will not fail.

**Alternatives considered**
* None

**Testing steps**
* Install site and verify that maps are working correctly if ENV variable for MAPS API is set.

**Merge requirements**
